index.html

```diff
@@ -20,10 +20,16 @@
         <li class="cellular"><img src="assets/Bars/Status Bars/Cellular Connectionmobile-top.png"></li>
         <li class="wifi"><img src="assets/Wifiwifi.png"></li>
         <li class="battery"><img src="assets/Bars/Status Bars/Batterymobile-top.png"></li>
+      </div><br><br>
+      <div class="nav">
+        <h1 class="my-logo">My logo</h1>
```

Comment:
- [ ] I would kindly suggest you make this an `anchor tag`. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 👍 👍